components/com_users/src/Model/ResetModel.php

```diff
@@ -527,7 +527,7 @@ public function checkResetLimit($user)
         $resetHours = (int) $params->get('reset_time');
         $result     = true;
 
-        $lastResetTime       = strtotime($user->lastResetTime) ?: 0;
+        $lastResetTime       = strtotime($user->lastResetTime ?? '') ?: 0;
```

Comment:
```suggestion
        $lastResetTime       = $user->lastResetTime === null ? 0 : strtotime($user->lastResetTime);
```

Better not calling `strtotime` when lastResetTime is null.

Reply:
yes